packages/web/src/components/notification/Notifications/UserProfilePictureList.tsx

```diff
@@ -14,25 +14,41 @@ const messages = {
 }
 
 type UserProfileListProps = {
-  users: User[]
+  users: Array<User>
   userIds: ID[]
+  limit?: number
+  totalOverride?: number
```

Comment:
nit: `totalOverride` is ambiguous - `userCount` or `totalUserCount` I think would be more descriptive. Could also use a comment clarifying that it defaults to the count of `userIds`

Reply:
i think we still need to somehow have the word override in there. user count and total user count may make it seem like they're always different numbers, but they're not necessarily.
maybe we can say totalUserCount instead of using the userIds length, and also have a totalUserCountOverride

Reply:
we actually just need total user count, the user ids are only used for that and nothing else, so that one prop is enough